orttraining/orttraining/test/training_api/trainer/trainer.cc

```diff
@@ -255,10 +255,10 @@ int RunTraining(const TestRunnerParameters& params) {
       &session));
 
   size_t train_mode_output_count, eval_mode_output_count = 0;
-  ORT_RETURN_ON_ERROR(g_ort_training_api->TrainingSessionGetTrainModeOutputCount(session, &train_mode_output_count));
+  ORT_RETURN_ON_ERROR(g_ort_training_api->TrainingSessionGetTrainingModelOutputCount(session, &train_mode_output_count));
```

Comment:
Just FYI - I intentionally named this as TrainMode and not TraingingModel because the fact that we have multiple models for training and eval is really an implementational detail.